com.unity.formats.alembic/Editor/Importer/AlembicStreamPlayerEditor.cs

```diff
@@ -97,8 +122,80 @@ public override void OnInspectorGUI()
             }
 
             EditorGUI.EndDisabledGroup();
+            if (!serializedObject.isEditingMultipleObjects)
+            {
+                DrawStreamSettings(streamPlayer);
+            }
+
+            serializedObject.ApplyModifiedProperties();
+        }
+
+        static void DrawStreamSettings(AlembicStreamPlayer player)
+        {
+            using (var check = new EditorGUI.ChangeCheckScope())
+            {
+                // Draw the stream proeprties
+                var pathSettings = "streamSettings.";
+
+                var so = new SerializedObject(player.StreamDescriptor);
+                so.Update();
+                var settings = so.FindProperty("settings");
+
+                EditorGUILayout.LabelField("Scene", EditorStyles.boldLabel);
+                {
+                    EditorGUI.indentLevel++;
+                    EditorGUILayout.PropertyField(settings.FindPropertyRelative("scaleFactor"),
+                        new GUIContent("Scale Factor",
+                            "How much to scale the models compared to what is in the source file."));
```

Comment:
Suggestion (tooltip):
`Use this property to resize models relative to their dimensions in the source file.`